src/styles/generic/_select.scss

```diff
@@ -11,6 +11,14 @@ select {
   padding-left: 4px;
   width: 100%;
 
+  &.w-select--borderless {
+    border: 0;
+
+    &:focus {
+      border: 1px solid $WEB_PRIMARY_COLOR;
```

Comment:
ah just noticed that the border is causing the text to shift:
![Kapture 2020-06-22 at 9 46 58](https://user-images.githubusercontent.com/1066253/85313741-55ac1f80-b46d-11ea-8aab-6b0a734a7089.gif)
You might try `outline` or `box-shadow` to see if either of those work better.